wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/BackupLocalStore/BackupLocalStore+Messages.swift

```diff
@@ -39,7 +39,7 @@ extension BackupLocalStore {
             fetchRequest.propertiesToFetch = ["nonce_data"]
 
             let messages = try backupContext.fetch(fetchRequest) as! [ZMMessage]
-            return Set(messages.compactMap(\.nonce).map(\.uuidString))
+            return Set(messages.compactMap(\.nonce).map { $0.transportString() })
```

Comment:
thought: I wonder if we can have a type for messageNonce so comparing nonce hides the details of id being lowercase or uppercase

Reply:
It's a core data attribute so it needs to stay as a UUID. Using a custom type would likely mean we'd need to make the nonce property private and encapsulate it in the custom type. So essentially might result in a couple of changes in different places to get it to be consistently adopted. But I like the thought. 

Also I guess it doesn't strictly need to be used everywhere and we could introduce it in the backup context